lib/forklift/box_distributor.rb

```diff
@@ -217,16 +219,7 @@ def configure_virtualbox(machine, box)
         p.memory = box.fetch('memory').to_i * @settings['scale_memory'].to_i if box.fetch('memory', false)
 
         bridged = box.fetch('bridged', false)
-
-        if bridged
-          override.vm.network :public_network, bridge: bridged
```

Comment:
This line will no longer be there. Is that intentional?

Reply:
nope!